packages/calcite-components/src/components/handle/handle.e2e.ts

```diff
@@ -125,4 +125,14 @@ describe("calcite-handle", () => {
   describe("translation support", () => {
     t9n("calcite-handle");
   });
+
+  it("sets application role on handle", async () => {
```

Comment:
Similar to adding context in the host, could you add a bit more detail to the test name? I think we should add more info to these types of a11y tests (role/ARIA role).